json/json-derivation/src/main/java/io/sphere/json/annotations/JSONTypeHintField.java

```diff
@@ -13,4 +13,5 @@
 @Target({TYPE})
 public @interface JSONTypeHintField {
     String value() default "type";
-}
\ No newline at end of file
+    String defaultType() default "";
```

Comment:
I guess you though about using `null` vs `""`. What arguments put in the balance of using `""`?